deploy/charts/cert-manager/templates/webhook-mutating-webhook.yaml

```diff
@@ -33,21 +27,15 @@ webhooks:
           - UPDATE
         resources:
           - "*/*"
-    {{- if $isV1AdmissionRegistration }}
     admissionReviewVersions: ["v1", "v1beta1"]
     timeoutSeconds: {{ .Values.webhook.timeoutSeconds }}
-    {{- end }}
     failurePolicy: Fail
-{{- if (semverCompare ">=1.12-0" .Capabilities.KubeVersion.GitVersion) }}
     # Only include 'sideEffects' field in Kubernetes 1.12+
     sideEffects: None
-{{- end }}
     clientConfig:
-{{- if (semverCompare "<=1.12-0" .Capabilities.KubeVersion.GitVersion) }}
       # Set caBundle to empty to avoid https://github.com/kubernetes/kubernetes/pull/70138
       # in Kubernetes 1.12 and below.
       caBundle: ""
-{{- end }}
```

Comment:
It looks like this whole block is part of the original hack: https://github.com/jetstack/cert-manager/pull/2664/files

So perhaps remove the empty caBundle too.

Reply:
True.... sorry i misread the code here!